book/src/getting_started.md

```diff
@@ -4,7 +4,7 @@ First, add the following to your crate's `Cargo.toml`:
 
 ```toml
 # in section [dependencies]
-rinja = "0.12.1"
+rinja = "0.3.5"
```

Comment:
Didn't we replace all this with `askama`? O.o

Reply:
Look at the destination branch. :) This PR is for rinja 0.3.6 release containing a deprecation notice.

Reply:
Oh boy, sorry. ^^'